### ISSUE TITLE
Add BUILD.bazel files

### ISSUE THREAD

**Issue by zombiezen** [capnproto/go-capnproto2#88]
I'm using Cap'n Proto more and more with [Bazel](https://bazel.build/) and it would be nice to have support directly in this repository (instead of hacking it [like I did with mcm](https://github.com/zombiezen/mcm/tree/de340cff88a12cd893141a2a1ca422c3b9e099d5/third_party/golang/capnproto)).

Specifically out of scope for this bug is having the `capnp_library` rules here.  I'd rather do that in the main C++ repository.

This is blocked on #83.
